_posts/2020-09-20-pulp-3.7.0-is-generally-available.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Pulp 3.7.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - release
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.7.0!
+
+
+## Export enhancements
+
+There have been a number of enhancements to the export process:
+
+* A progress report has been added to export tasks. [#6541](https://pulp.plan.io/issues/6541)
+
+* Improved handling of failed exports. Previously, when an export failed, the export's tarball was left on the filesystem. This has now been cleaned up so that nothing remains and there is a clean slate. [#7246](https://pulp.plan.io/issues/7246)
+
+* A `TOC_info` file has been added to point to the location of the TOC file during the export process. [#7221](https://pulp.plan.io/issues/7221)
```

Comment:
I would probably spell out what TOC stands for especially for non-native English speakers.

Reply:
I'm a whole week out of docs and breaking cardinal rules. Cheers :)